spring-integration-core/src/main/java/org/springframework/integration/codec/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @author David Turanski
+ */
+package org.springframework.integration.codec;
```

Comment:
Another rule:
New line in the end of each file.

From other side this `package-info.java` must have JavaDocs.
That's is its premise :smile: